Scripts/today.js

```diff
@@ -0,0 +1,28 @@
+window.addEventListener( 'load', loadSite)
+
+function loadSite() {
+
+    const d = new Date();
+    document.querySelector(".year").innerHTML = d.getFullYear();
+
+    //change 0-11 to January-December
+    document.querySelector(".month").innerHTML = d.getMonth();
+
+    document.querySelector(".day").innerHTML = d.getDate();
+
+    //change 0-9 to 00-09
+    document.querySelector(".hour").innerHTML = d.getHours() +" :";
+
+    //change 0-9 to 00-09
+    document.querySelector(".minute").innerHTML = d.getMinutes() +" :";
+
+    //change 0-9 to 00-09
```

Comment:
Perhaps we don't need to show the seconds

Reply:
Okey, we remove it in the next stage.